packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -172,6 +172,7 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
     // If we are moving items within the same picker, remove them from their old places as well
     if (draggedIndex > -1) {
       indicesToRemove = focusedItemIndices.includes(draggedIndex) ? [...focusedItemIndices] : [draggedIndex];
+      setDraggedIndex(-1);
```

Comment:
This shouldn't be necessary, as we're clearing this in onDragEnd, which we should be receiving in this case. Clearly that's not always happening, but I wonder why?

Reply:
For some reason, I only get _onDragEnd when dragging BETWEEN wells. I don't see that called with dragging WITHIN a single well.

Reply:
Deleted this line, and changed the fix approach.